_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,328 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are instruments to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML element][] that has a [visible][] [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed:** the `innerText` property of the [element][html element] changes multiple times within a 10 minute time span where there is no [user interaction][]; and
+- **no child changed:** the [element][html element] does not have [children][child] in the [flat tree][] whose `innerText` property also changes; and
+- **not alone:** the [element][html element] has an [ancestor][] element in the [flat tree][] with a non-empty `innerText` property whose value is different from the `innerText` of the test target.
+
+**Note:** The test targets for this rule are elements that have their text content change multiple times, on a page with further content, if that change happens without user intervention.
+
+**Note:** The 10 minute time span is an arbitrary limit which is not included in WCAG. Content that updates less frequently, or that doesn't update on a regular interval, will be inapplicable for this rule but may nonetheless fail [Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]. The accessibility problem tends to be less severe for longer time periods, and without a time limit, testing this rule consistently would be impractical.
+
+## Expectation 1
+
+For each test target there exists a set of [instruments][instrument] to achieve one of the following objectives:
+
+- pause and resume the change of the [visible text content][]; or
+- stop the change of the [visible text content][]; or
+- hide the content that changes; or
+- alter the frequency of the changes of the [visible text content][].
+
+**Note:** If there is more than one test target, the same [instrument][] may be used to pause, stop, hide or alter the frequency of several or even all test targets.
+
+## Expectation 2
+
+For each [instrument][] in the set of [instruments][instrument] from Expectation 1, one of the following is true:
+
+- the [instrument][] is in the same [web page](#web-page-html) of the test target; or
+- instructions for the user to locate the [instrument][] are in the same [web page](#web-page-html) of the test target.
```

Comment:
Here's how far I got with an alternative. It definitely needs more work, taking out all the ambiguities, but this is roughly the direction I'm thinking:


> The instrument from expectation 1 can be found in a [clearly labelled location][]
> 
> # Clearly labelled location
> 
> Secondary information and alternative controls of functionality is often not displayed together with primary information or functionality. For example an option to change a web page to dark mode may be place on an options page, instead of being available in every page and page state of a website. Another example is a maps app, where instead of using GPS, an option is availble in a dropdown menu to set the current location of the device. Such content is (usually) placed in a clearly labelled location.
> 
> The location of some content can be said to be "clearly labelled" when it can be found by activating user interface components who's label indicates that the purpose of it may lead to finding the content, 
or to some further page or page state from which this action can be repeated until the content is found.
> 
> Whether or not the content is "clearly labelled" depends on the starting point of the search. If a page A has a clearly labelled link to some piece of important content, it is clearly labelled. Page B, which can be in the same website may not have such a link, or may have a link with an ambiguous link text, and so the location of the content starting from page B is not clearly labelled.

Reply:
@WilcoFiers new definition pushed, together with updated expectation. Can you please review again?